opengever/base/error_log.py

```diff
@@ -0,0 +1,105 @@
+from copy import deepcopy
+from datetime import datetime
+from opengever.base.interfaces import IBaseSettings
+from opengever.base.utils import to_safe_html
+from opengever.redis.client import redis_client_manager
+from plone import api
+import json
+
+
+def is_user_visible_error_logs_feature_enabled():
+    return api.portal.get_registry_record(
+        'is_user_visible_error_logs_feature_enabled', interface=IBaseSettings)
+
+
+class ErrorLogRedis:
```

Comment:
```suggestion
class RedisErrorLog:
```

`ErrorLogRedis` tönt nach Yoda Style 😉

Reply:
😂